src/content/it/fundamentals/payments/android-pay.md

```diff
@@ -0,0 +1,510 @@
+project_path: /web/fundamentals/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: Android Pay consente acquisti online semplici e sicuri ed elimina
```

Comment:
description should be on a single line, no line-breaks in the description.

Reply:
@lucab85 I have created a pull request to fix the issue @petele mention ↑. Please check it out. https://github.com/lucab85/WebFundamentals/pull/1